libpod/container.go

```diff
@@ -1173,6 +1173,24 @@ func (c *Container) Networks() ([]string, bool, error) {
 	return c.networks()
 }
 
+// NetworkMode gets the configured network mode for the container.
+// Get actual value from the database
+func (c *Container) NetworkMode() string {
```

Comment:
I think you copied the logic from inspect. It would make sense to use the complete inspect logic here and use this function in inspect to prevent code duplication.

Reply:
@Luap99 thanks this is resolved in latest commit.